pkg/target/manager.go

```diff
@@ -403,46 +446,41 @@ func (m *managerImpl) ShootClusterClient(ctx context.Context, t Target) (client.
 	// the shoot, which is also needed because we need to locate the kubeconfig
 	// on disk.
```

Comment:
```go
	// [...] which is also needed because we need to locate the kubeconfig
	// on disk.
```
is this still true?

Reply:
I don't know exactly but from my point of view it is not true anymore. The origin is the `kubeconfig_cache` which anyway should be removed. The same error is thrown there. I will remove that check here.